src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/test/RandomTestData.kt

```diff
@@ -35,6 +35,8 @@ fun randomName(): String = randomLowerCaseString()
 fun randomNationalInsuranceNumber(): String = UUID.randomUUID().toString()
 fun randomDriverLicenseNumber(): String = UUID.randomUUID().toString()
 
+fun randomFullAddress(): String = UUID.randomUUID().toString()
```

Comment:
Can we have something which looks a bit more like an address? Maybe a random number a few random strings and a postcode?